receiver/gitproviderreceiver/internal/scraper/githubscraper/github_scraper.go

```diff
@@ -29,6 +29,20 @@ type githubScraper struct {
 	rb       *metadata.ResourceBuilder
 }
 
+// Adding this so that the returns from getting repositories from a search query vs a team query can be
+// put into one struct and the loop for processing the data can be the same
+// The underlying structure of the repository nodes are the same but genqlient generates them uniquely
+// for each query hence this structure.  Probably a better way, just want it to work for now.
```

Comment:
Let's find the better way now instead of leaving this comment in here as is.

Reply:
https://github.com/Khan/genqlient/blob/main/docs/operations.md#sharing-types

Forgot fragments exist